bring/src/main/java/com/bobocode/petros/bring/scanner/impl/DefaultBeanPostProcessorScanner.java

```diff
@@ -17,8 +19,19 @@ public class DefaultBeanPostProcessorScanner implements BeanPostProcessorScanner
      * @param classes classes to scan
      * @return list of bean post processors
      */
+
     @Override
     public List<BeanPostProcessor> scan(Set<Class<?>> classes) {
-        return null;
+        return classes.stream()
+                .flatMap(Stream::ofNullable)
```

Comment:
As far as I understand, here a non null element is taken and put into a new stream (so we have a stream of a single element) or otherwise it returns an empty stream. Could you please check whether this line can be replaced with .filter(Objects::nonNull) and we still have the same behavior? meaning, that if there are some elements in a given set they'll be processed or otherwise an empty list is returned.